blockchain/types/account/doc.go

```diff
@@ -14,5 +14,41 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package account contains types related to Klaytn accounts.
+/*
+Package account implements Account used in Klaytn.
+
+Inside the package, the type, structure, functions, and interfaces associated with the Account are defined.
+
+
+Type of Account
+
+There are three types of Accounts used in Klaytn.
+
+  - LegacyAccountType
+  - ExternallyOwnedAccountType
+  - SmartContractAccountType
+
+AccountCommon implements the structure and functions common to Klaytn Account, which also implements the Account interface.
+
+EOA(ExternallyOwnedAccount) and SCA(SmartContractAccount) are implemented in a structure that includes AccountCommon.
+
+LegacyAccount is implemented separately according to the account interface.
+
+
+
+Source Files
+
+Account related functions and variables are defined in the files listed below
```

Comment:
Please, mark the period at the end of line.

Reply:
Period is added :) Thank you for reviewing